flycheck.el

```diff
@@ -7439,6 +7500,17 @@ See URL `https://github.com/eslint/eslint'."
             (option-list "--rulesdir" flycheck-eslint-rules-directories)
             "--stdin" "--stdin-filename" source-original)
   :standard-input t
+  :enabled (lambda (checker)
+             (let* ((command (concat (flycheck-find-checker-executable checker)
+                                     (if flycheck-eslintrc
+                                         (concat " --config " flycheck-eslintrc)
+                                       "")
+                                     " --print-config "
+                                     (flycheck-compute-working-directory checker)
+                                     "."))
+                    (exitcode (call-process-shell-command command nil nil nil)))
+               (cond ((eq exitcode 0) nil)
+                     (t "Configuration not found"))))
```

Comment:
One small addition: this code isn't safe wrt spaces in working directory or file name.  I think this code might be able to use `flycheck-substitute-argument`, and if not, `shell-quote-argument`